webapp/src/components/VideoSection/VideoSection.js

```diff
@@ -5,7 +5,7 @@ const Video = ({ ...pros }) => (
   <iframe
     title={'unique' + Math.random()}
     width="100%"
-    height="100%"
+    height="400px"
```

Comment:
What if we use viewport height (vh) instead of px? @dloynaz

Reply:
It seems not to respond correctly to vh. My understanding is that a 20vh represents 20% of the viewport. In this case I had to put it on 600vh to notice a difference. It seems to me that pixels are more stable if we are going to work with iframe. I think that the Box component is causing the shrink. Something that has happened to me before working with flex is that if the assets do not have a specific resolution, they will be imposed by the parent component (in case you are using flex). Try to correct this by using a flex: 2 or flex: 3 on the video component but it seems to be unresponsive. So I finally ended up  using pixels. However, it seems to respond well to different screen sizes. I have tried using a MaterialUi mediacard but the behavior seems to be the same. In any case we could also use a ReactPlayer component. It is the one that I normally use when working with video in React.

Reply:
Got it. Thank you for the explanations. Looks great!